maintainers/maintainer-list.nix

```diff
@@ -58,6 +58,12 @@
     See `./scripts/check-maintainer-github-handles.sh` for an example on how to work with this data.
 */
 {
+  _0b11stan = {
+    name = "Tristan Auvinet Pinaudeau";
+    email = "tristan@tic.sh";
+    github = "0b11stan";
+    githubId = 27831931;
+  };
```

Comment:
this should be it's own commit called: `maintainers: add _0b11stan`

Reply:
So sorry, I missed the section in CONTRIBUTING and thought it was not mandatory anymore, I fixed it.